tile_generator/config_unittest.py

```diff
@@ -150,12 +150,11 @@ def new_comparer(expected, given, path):
 			else:
 				self.assertEquals(expected, given, (path, 'Expected to have the value:\n%s\nHowever, instead got:\n%s' % (expected, given)))
 
-
 		for release in cfg['releases']:
-                    if release.has_key('consumes_for_deployment'):
-                        self.assertDictEqual({'from': 'redis'}, release['consumes_for_deployment']['redis'])
-                        # Remove it so it is compatible with the OLD expected output
-                        release['consumes_for_deployment'].pop('redis')
+			if release.has_key('consumes_for_deployment'):
```

Comment:
Unnecessary indentation here with no changes

Reply:
Change is to make indentation consistent with surrounding blocks of code.